Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -1687,6 +1691,16 @@ def prepare_PATGEN(self, sequence = "miniGEN"):
             raise Exception("PATGEN step can only run on MC")
         return
 
+    def prepare_NANO(self, sequence = "nanoAOD"):
```

Comment:
if this runs already, perhaps a step can be added to a standard short matrix workflow
so that the code is tested regularly.

Reply:
are the externals already in the IB?

Reply:
the cmsDriver integration works already in this PR, but I'm a bit lost at where and how to insert it into PyReleaseValidation for the matrix (for the standard configuration, we probably want to run it on top of the miniAOD produced by the same 94X release)

Reply:
Makes sense to do that yes - this should just be a new step added to ~all run2 workflows that already produce miniAOD output. (relval_steps.py is the fun place to start)